platform_db_backup.sh

```diff
@@ -41,6 +41,12 @@ if [[ -v SFTP_USERNAME ]]; then
     echo "${SSH_PUBLIC_KEY}" > ~/.ssh/id_rsa.pub
     chmod 600 ~/.ssh/id_rsa
     chmod 600 ~/.ssh/id_rsa.pub
+
+   ssh-keygen -f ~/.ssh/id_rsa.pub -e -m PKCS8 > ~/.ssh/id_rsa.pem.pub
+    if [ ! -f ".ssh/secret.key" ]; then
+      echo "Generating random secret key."
+      openssl rand -out ~/.ssh/secret.key 32
```

Comment:
How do you get a key if it doesn't exist yet? you have to ssh to the server and copy it from there - correct?

Reply:
No, the SSH key is saved in SSH_SECRET_KEY and SSH_PUBLIC_KEY variables.

Reply:
ssh key yes, but I was referring to the secreet with which the db file is encrypted. It might be ok to have a possibility to add it via env var.

Reply:
removed this and the script is now using a password passed through a env var.